app/Services/Post.php

```diff
@@ -153,6 +154,17 @@ public static function getPostFromUrl( $postUrl ) {
 		// Create the ACF Widgets from DOM content
 		$widgets = Widget::getWidgetsFromDom( $postDom );
 
+		// if there is a video header, convert to embed widget
+		if ($hero == "hero-without-media") {
+			$headerEmbed = Video::getVideoFromHeader($postObject->provider, $postObject->videoId);
+			array_unshift($widgets, $headerEmbed);
+			/**
+			* @var $_logger \Croissant\DI\Dependency\CatfishLogger
+			*/
+			$_logger = \Croissant\App::get( \Croissant\DI\Interfaces\CatfishLogger::class );
+			$_logger->notice("provider: " . $postObject->provider . ", video id: " . $postObject->videoId . ", url: " . $postUrl);
+		}
```

Comment:
This will not work unfortunately. Currently only errors are logged to file and notice have lower status. You can create another logger (recommended) or change notice to error (in this case url will be mixed with other debug code)